triggers/run-completion-event-trigger/proto/run_completion_event_trigger.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package run_completion_event_trigger;
+
+option go_package = "github.com/sky-uk/kfp-operator/triggers/run-completion-event-trigger/proto/run-completion-event-trigger";
+
+service RunCompletionEventTrigger {
+  rpc ProcessEventFeed(RunCompletionEvent) returns (google.protobuf.Empty) {};
+}
+
+message RunCompletionEvent {
+  string pipeline_name = 1;
+  string provider = 2;
+  string run_configuration_name = 3;
+  string run_id = 4;
+  string run_name = 5;
+  repeated ServingModelArtifact serving_model_artifacts = 6;
+  string status = 7;
```

Comment:
we really going to use a string? Is there no exhaustive list of status coming back that we can check against